node/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change log
 
+## v0.3.1
+
+### Non-breaking changes
```

Comment:
[🤔 ] breaking changes…？

Reply:
It should be treated as a breaking change, as the users of this package **have to** replace their peer dependency.
Could you move this to the (newly-created) v0.4.0 section?